lib/net.c

```diff
@@ -41,10 +45,14 @@
  */
 #define GATEKEEPER_RETA_MAX_SIZE (ETH_RSS_RETA_SIZE_512 / RTE_RETA_GROUP_SIZE)
 
+#define GATEKEEPER_IPV4 (1)
+#define GATEKEEPER_IPV6 (2)
```

Comment:
These names are too generical to help readability in code. Suggestions: GK_CONFIGURED_IPV4 and GK_CONFIGURED_IPV6.